app/views/hyrax/admin/collection_types/_form.html.erb

```diff
@@ -15,13 +15,23 @@
   <% form_url = @form.id.nil? ? hyrax.admin_collection_types_path : hyrax.admin_collection_type_path(@form.id) %>
   <%= simple_form_for @form, url: form_url, as: :collection_type do |f| %>
     <div class="tab-content">
-      <div id="metadata" class="tab-pane active">
-        <div class="panel panel-default labels">
-          <div class="panel-body">
-            <%= render 'form_metadata', f: f %>
+      <% if @collection_type.machine_id == "admin_set" %>
```

Comment:
Push this check to the collection type model. In the collection type model, we should the method `@collection_type.admin_set?`. This way we have a consistent mechanism for asking a collection type if it is an admin_set (which has specific behavior in the current implementation).